src/database/pair-liquidity-info-history.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from './prisma.service';
+import { PairLiquidityInfoHistory } from '@prisma/client';
+
+@Injectable()
+export class PairLiquidityInfoHistoryService {
+  constructor(private prisma: PrismaService) {}
+
+  getLastSyncedHeight(pairId: number) {
```

Comment:
should we add limit 1 here?

Reply:
I think Prisma's `findFirst` already boils down to that.